src/main.py

```diff
@@ -223,45 +215,46 @@ def main():
                                  value_serializer=lambda v: json.dumps(v).encode('utf-8'))
 
         for msg in consumer:
-            try:
-                rec = eval(msg.value)
-                timestamp = rec['timestamp']
-                ftr_vector = rec['ftr_vector']
-                measurement = ftr_vector[0] # first feature is the target measurement
+            # try:
```

Comment:
try-catch was there for a reason ... probably because of potential crash due to misformatted messages. Please add it back.

Reply:
Yeah, that was an oversight. I wanted full error stack while debugging so I just put the try block in comments and forgot to uncomment after will fix it.